presto-docs/src/main/sphinx/release/release-0.140.rst

```diff
@@ -12,11 +12,14 @@ General Changes
 * Fix race which can cause queries to fail with a ``REMOTE_TASK_ERROR``.
 * Optimize :func:`array_distinct` for ``array<bigint>``.
 * Optimize ``>`` operator for :ref:`array_type`.
+* Fix projection push down to not duplicate non-deterministic functions
```

Comment:
These are not very meaningful to users. I'd rephrase as:
* Fix an optimization issue that could result in non-deterministic functions being evaluated more than once and produce unexpected results.